JConvex/gap/ExternalPolymakeCone.gi

```diff
@@ -206,9 +242,7 @@ InstallMethod( Polymake_CanonicalConeFromInequalities,
         od;
         
         # compute linear span
-        command_string := Concatenation( Polymake_H_Rep_command_string( cone ), ".LINEAR_SPAN" );
-        JuliaEvalString( command_string );
-        s := JuliaToGAP( IsString, Julia.string( Julia.ConeByGAP4PackageConvex ) );
+        s := JuliaToGAP( IsString, Julia.string( cone!.pmobj.LINEAR_SPAN ) );
```

Comment:
of course converting this data into a string and then parsing that string (with `EvalString` no less!) also should be replaced by simple and more efficient code. I've started on that but now it's weekend time, so I'll do it later. For now, this PR should be a big step forward on its own

Reply:
Nevermind, it was trivial so I just did it ;-).

Only two uses of `EvalString` are left, both should still be replaced, but now I'll really quit :-)